From 3fa30bf7004bb8069826ef85487f4753666a73da Mon Sep 17 00:00:00 2001
From: Tomasz Duszynski <tduszyns@gmail.com>
Date: Sat, 2 Feb 2019 10:00:03 +0100
Subject: [PATCH] iio: chemical: sps30: remove printk format specifier

pr_fmt is used by printk wrappers. There are not any in the driver
code so remove the format specifier.

Signed-off-by: Tomasz Duszynski <tduszyns@gmail.com>
Signed-off-by: Jonathan Cameron <Jonathan.Cameron@huawei.com>
---
 drivers/iio/chemical/sps30.c | 2 --
 1 file changed, 2 deletions(-)

--- a/drivers/iio/chemical/sps30.c
+++ b/drivers/iio/chemical/sps30.c
@@ -7,8 +7,6 @@
  * I2C slave address: 0x69
  */
 
-#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
-
 #include <asm/unaligned.h>
 #include <linux/crc8.h>
 #include <linux/delay.h>
